Outdated nrf_security library documentation for nRF52840 crypto cell 310 support

It appears the [documentation](https://github.com/openthread/openthread/tree/master/examples/platforms/nrf528xx/nrf52840#cryptocell-310-support) for cryptocell support with external applications is outdated. The library `libmbedcrypto_cc310_backend.a` no longer exists in `third_party/NordicSemiconductor/libraries/nrf_security/lib` among others.

The current contents of `third_party/NordicSemiconductor/libraries/nrf_security/lib` are:
```
libmbedcrypto_cc3xx.a   libmbedcrypto_shared.a     libmbedtls_tls_vanilla.a   libnrf_cc310_platform_0.9.4.a
libmbedcrypto_oberon.a  libmbedtls_base_vanilla.a  libmbedtls_x509_vanilla.a
```

How has the configuration and library includes changed for building external apps with cryptocell support? I tried including all the new libraries, sans the oberon one, and my builds are failing with many undefined mbedtls reference errors like the below:
```
libmbedtls_base_vanilla.a(ctr_drbg.c.obj): in function `mbedtls_ctr_drbg_random_with_add':
/home/map5/repo/ncs/mbedtls/library/ctr_drbg.c:564: undefined reference to `mbedtls_aes_crypt_ecb'
```

--- a/examples/platforms/nrf528xx/nrf52840/README.md
+++ b/examples/platforms/nrf528xx/nrf52840/README.md
@@ -135,11 +135,13 @@ mbedtls_platform_setup(NULL);
 When building an external application with OpenThread libraries and CryptoCell 310 hardware acceleration, use the following configuration:
 
 - Crypto libraries:
-  - `third_party/NordicSemiconductor/libraries/nrf_security/lib/libmbedcrypto_cc310_backend.a`
-  - `third_party/NordicSemiconductor/libraries/nrf_security/lib/libmbedtls_tls_vanilla.a`
-  - `third_party/NordicSemiconductor/libraries/nrf_security/lib/libmbedtls_x509_vanilla.a`
-  - `third_party/NordicSemiconductor/libraries/nrf_security/lib/libmbedtls_base_vanilla.a`
-  - `third_party/NordicSemiconductor/libraries/nrf_security/lib/libnrf_cc310_platform_0.9.2.a`
+  - `/third_party/NordicSemiconductor/libraries/nrf_security/lib/libmbedcrypto_shared.a`
+  - `/third_party/NordicSemiconductor/libraries/nrf_security/lib/libmbedtls_tls_vanilla.a`
+  - `/third_party/NordicSemiconductor/libraries/nrf_security/lib/libmbedtls_x509_vanilla.a`
+  - `/third_party/NordicSemiconductor/libraries/nrf_security/lib/libmbedcrypto_cc3xx.a`
+  - `/third_party/NordicSemiconductor/libraries/nrf_security/lib/libnrf_cc310_platform_0.9.4.a`
+  - `/third_party/NordicSemiconductor/libraries/nrf_security/lib/libmbedcrypto_oberon.a`
+  - `/third_party/NordicSemiconductor/libraries/nrf_security/lib/libmbedtls_base_vanilla.a`
 - Include directories:
   - `third_party/NordicSemiconductor/libraries/nrf_security/mbedtls_platform_config`
   - `third_party/NordicSemiconductor/libraries/nrf_security/include`

